scripts/release.js

```diff
@@ -20,29 +22,57 @@ const packageJson = require('../package.json');
 
 const {showChangelog, getChangelog, updateChangelog} = require('./lib/conventionalChangelog.js');
 
-shell.echo(`Algoliasearch-Helper release script`);
+const {canary: isCanary} = argv;
+
+if (isCanary) {
+  releaseCanaryVersion();
+} else {
+  releaseStableVersion();
+}
 
-checkEnvironment();
-mergeDevIntoMaster();
-showChangelog(shell);
-promptVersion(packageJson.version, (version) => {
-  bumpVersion(version, () => {
+function releaseCanaryVersion() {
+  shell.echo(`Algoliasearch-Helper release CANARY version`);
+
+  checkCleanWorkdir();
+  updateCanaryVersion(canaryVersion => {
     build();
-    updateChangelog(shell);
-    commitNewFiles(version);
-    publish();
-    goBackToDevelop();
+    publishOnNpm('canary');
+    revertStableVersion(packageJson.version, () => {
+      shell.echo(`Algoliasearch-Helper v${canaryVersion} released`);
+    });
   });
-});
+}
 
-function checkEnvironment() {
+function releaseStableVersion() {
+  shell.echo(`Algoliasearch-Helper release STABLE version`);
+
+  checkDevelopBranch();
+  checkCleanWorkdir();
```

Comment:
Those two functions are the equivalent of the previous call `checkEnvironment()`.